### ISSUE TITLE
"workaround for a bug in PGF 2.00": still relevant?

### ISSUE THREAD

**Issue by dbitouze** [pgf-tikz/pgfplots#349]
In the documentation of `pgfplots` version 1.16, a "workaround for a bug in PGF 2.00" is mentionned at page 273 and (2 times) at page 312: "something like `exp(-10^2)` must be written as `exp(0-10^2)"`.

AFAICS, this workaround isn't relevant anymore.

**Comment by hmenke** [comment]
Have you confirmed that this workaround is not necessary anymore when using current pgfplots with PGF 2.00?

**Comment by dbitouze** [comment]
>Have you confirmed that this workaround is not necessary anymore when using current `pgfplots` with `PGF` 2.00?

@hmenke  No. But I guess current `pgfplots` is mostly used with current `PGF`. Hence examples in the current documentation of `pgfplots` shouldn't use this workaround (but possibly mention it as comments for the rare cases where current `pgfplots` is used with `PGF` 2.00).

**Comment by hmenke** [comment]
> But I guess current pgfplots is mostly used with current PGF.

There is no PGF version check performed inside pgfplots. Instead pgfplots chooses to be perfectly backwards compatible to older PGF versions.

**Comment by dbitouze** [comment]
That's the reason I suggest for the current documentation of `pgfplots`:

- to let the examples use the current possible syntax of `PGF`,
- to _mention this workaround as comments_ (for the rare cases where current `pgfplots` is used with `PGF` 2.00).
